IPython/core/pylabtools.py

```diff
@@ -431,3 +466,10 @@ def configure_inline_support(shell, backend):
     )
 
     configure_inline_support_orig(shell, backend)
+
+
+def _matplotlib_manages_backends():
+    import matplotlib
+
+    mpl_version_info = getattr(matplotlib, "__version_info__", (0, 0))
+    return mpl_version_info >= (3, 9)
```

Comment:
:+1:

Reply:
As I mentioned in https://github.com/ipython/ipython/pull/14371#discussion_r1555393766 above, I am inclined to change this check to use `hasattr` rather than version number.